examples/full/clickpipe/kafka_azure_eventhub/provider.tf

```diff
@@ -0,0 +1,23 @@
+terraform {
```

Comment:
Like we do in the other examples, please add a template file for the provider (see https://github.com/ClickHouse/terraform-provider-clickhouse/blob/clickpipe/examples/full/development/aws/provider.tf.template) this will allow automation to automatically bump examples to use latest release as part of the release automation